src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    operate(callback, value) {
+      if (typeof value === 'number') {
+        this.result = callback(value, this.result);
+
+        return this;
+      }
+    },
+    add(value, result) {
+      return result + value;
+    },
+    subtract(value, result) {
+      return result - value;
+    },
+    divide(value, result) {
+      if (value !== 0) {
+        return result / value;
+      }
+
+      return 'error: division by zero is not possible';
```

Comment:
Consider it will be better. You can read more here
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw
```suggestion
      return throw new Error('error: division by zero is not possible');
```

Reply:
Thank you for the new information :)

Reply:
![image](https://github.com/mate-academy/js_advanced_calculator/assets/135485575/7e28d4c3-91aa-40e7-87de-1d22af83e996)
but for some reason linter says problems...